test/e2e/e2e_test.go

```diff
@@ -242,15 +377,117 @@ var _ = Describe("Manager", Ordered, func() {
 
 		// +kubebuilder:scaffold:e2e-webhooks-checks
 
-		// TODO: Customize the e2e test suite with scenarios specific to your project.
-		// Consider applying sample/CR(s) and check their status and/or verifying
-		// the reconciliation by using the metrics, i.e.:
-		// metricsOutput := getMetricsOutput()
-		// Expect(metricsOutput).To(ContainSubstring(
-		//    fmt.Sprintf(`controller_runtime_reconcile_total{controller="%s",result="success"} 1`,
-		//    strings.ToLower(<Kind>),
-		// ))
+		It("creates multiple ModelService resources using kubectl", func() {
+			By("Creating the basic base configmap")
+			var cm corev1.ConfigMap
+			err := yaml.Unmarshal([]byte(configMapYAML), &cm)
+			Expect(err).NotTo(HaveOccurred())
+
+			cm.Namespace = "default"
+
+			err = k8sClient.Create(ctx, &cm)
+			Expect(err).NotTo(HaveOccurred())
+
+			By(fmt.Sprintf("Creating ModelService %s", modelServiceName))
+
+			modelService := &msv1alpha1.ModelService{
+				TypeMeta: metav1.TypeMeta{
+					APIVersion: msv1alpha1.GroupVersion.String(),
+					Kind:       "ModelService",
+				}, ObjectMeta: metav1.ObjectMeta{
+					Name:      modelServiceName,
+					Namespace: namespace,
```

Comment:
Should this be `namespace`? I think we want to deploy the modelservice in a different ns than `modelservice-system`.  Same problem probably several times.